FoodOrderingSystem.py

```diff
@@ -0,0 +1,925 @@
+        return login
+
+    def process_order(self, cust_id):
+        session = self.Session()
+        o = self.customer.create_order_id(cust_id, session)
+        selection = """
+        
+        0. Back
+        1. Add food to order
+        2. Remove food to order
+        3. Update food to order
+                
+        Select option: 
+
+        """
+        option = int(input(selection))
+
+        while option != 0:
+            
+            if option == CUST_OPT_ADD_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                food_qty = input("Enter food quantity: ")
+                add = self.customer.add_food_to_order(o.order_id, food_id, food_qty, session)
+                if add:
+                    print("Add successful!")
+                else:
+                    print("Add not successful. Please try again!")
+
+            elif option == CUST_OPT_REMOVE_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                remove = self.customer.remove_food_to_order(o.order_id, food_id, session)
+                if remove:
+                    print("Items removed")
+                else:
+                    print("Items not removed")
+
+            elif option == CUST_OPT_UPDATE_FOOD_TO_ORDER:
+                food_id = input("Enter food ID: ")
+                food_qty = input("Enter food quantity: ")
+                update = self.customer.update_food_to_order(o.order_id, food_id, food_qty, session)
+                if update:
+                    print("Items updated")
+                else:
+                    print("Items not updated")
+
+            option = int(input(selection))
+        
+        if o:
+            print("\nYour order is generated. \nOrder number: {}".format(o.order_id))
+        else:
+            print("Order not generated. Please try again!")
+
+    def view_order(self, order_id):
+        session = self.Session()
+        view_order_item = self.customer.view_order_per_item(order_id, session)
+        if view_order_item:
+            for fc, fd, cos in view_order_item:
+                print("\nFood category: {} \nFood name: {} \nFood price: {} \nFood quantity: {} \nTotal per item: {}".format(fc.name, fd.food_name, fd.price, cos.food_qty, (fd.price*cos.food_qty)))
+
+        view_order_grand = self.customer.view_order_grand_total(order_id, session)
+        if view_order_grand:
+            for fd, cos, cd, cosa, t in view_order_grand:
+                print("\nCustomer name: {} \nOrder ID: {} \nTotal bill: {}".format(cd.cust_name, cosa.order_id, t))
+
+    def checkout(self, order_id, order_status, order_address, checkout_time, estimated_time, bill_amount):
+        session = self.Session()
+        c = self.customer.checkout(order_id, order_status, order_address, checkout_time, estimated_time, bill_amount, session) 
+        if c:
+            print("Checkout sucessful!")
+        else:
+            print("Checkout not sucessful!") 
```

Comment:
These 4 lines can be reduced to a one liner:

```python
print("Checkout status: {}".format("Sucessful" if True else "Not Sucessfull"))
```

You can do the same for all the other places you have this if/else 4 line block.